### PR TITLE
New entry event

### PR DESCRIPTION
**Pull request by DannyWPWS**

### CODE REVIEW

**Comment by DannyWPWS** [comment]
Marking as not ready as it needs to go in after this one :) https://github.com/wpwhitesecurity/wsal-wpforms/pull/114

File: wp-security-audit-log/custom-sensors/WPFormsSensor.php [hunk 1]
```diff
@@ -60,6 +63,50 @@ public function get_before_post_edit_data( $post_id ) {
 		}
 	}
 
+	function event_entry_added( $fields, $entry, $form_data, $entry_id ) {
```
Comment (martinkrcho) [inline wp-security-audit-log/custom-sensors/WPFormsSensor.php:66]: Let's add some phpdoc for this function, please.

**Review by martinkrcho — CHANGES_REQUESTED** [review]
Overall changes look OK. Please see my comments. Only minor stuff again. Is the code passing the coding standards check for you? Couple of things should definitely fail.

**Review by martinkrcho — APPROVED** [review]
All changes are looking good now.

**Pull request merged**